### PR TITLE
payloads/external/iPXE/Makefile: build using $CPUS jobs

### PR DESCRIPTION
**Pull request by mkopec**
Allow using the $CPUS variable to control how many jobs are used for building iPXE. Consideralby speeds up compilation when building coreboot with `CPUS=17 NUM_CPUS=17 make -j17` on an i5-1240P (until now, iPXE took the longest to build of all Dasharo components)

### CODE REVIEW

File: payloads/external/iPXE/Makefile
Comment (krystian-hebel) [inline payloads/external/iPXE/Makefile:83]: Would it make sense to add this to `PXE_MAKE_OPTS` instead?

Comment (mkopec) [reply payloads/external/iPXE/Makefile]: yeah that makes more sense :) Fixed in dd0e7d42296f4ef0091af7d97a514f33006a68e8

**Review by krystian-hebel — APPROVED** [review]

**Comment by krystian-hebel** [comment]
I'm waiting for CI to finish building before merging.

**Pull request merged**
